client/src/containers/FirebaseApiSetUpTest/firebase/rememberMe.jsx

```diff
@@ -0,0 +1,12 @@
+import * as firebase from '../../../UserAuthUtilities/firebase';
+
+export function rememberMe(isChecked){
+  if(isChecked){
+    console.log("remember me is checked")
+    firebase.auth().setPersistence(firebase.auth.Auth.Persistence.LOCAL);
+  }
+  else{
+    console.log("remember me is not checked")
+    firebase.auth().setPersistence(firebase.auth.Auth.Persistence.SESSION);
+  }
+}
```

Comment:
I think it makes more sense to put the ```clicked``` state logic reversal in the input of the function call rather than invert the rememberMe() if conditions.